### DISCUSSION TITLE
Consider diffing against a SHA

### DISCUSSION THREAD

**Discussion by kboom** [sajalverma17/GitBranchDiffer#36] [Ideas]
Like in the visual studio code plugin (tree compare) it would be useful to be able to diff not only to branches, but also to specific git SHAs. Great work here!

**Comment by sajalverma17** [comment]
Thanks for the suggestion! I started looking into it a bit, the LibGit2Sharp library I'm using seems to be quite flexible. I'll try to do this when I find the time.
